modules/ListView.js

```diff
@@ -295,7 +306,6 @@
           // Shouldn't get here. Item view is missing...
```

Comment:
Should this throw an error so the developer knows what went wrong (that something went wrong)?